consistency/tool/comparison/comparison.py

```diff
@@ -1,32 +1,63 @@
+from strsimpy.sorensen_dice import SorensenDice
+import sys
+
+
 class Comparison:
-    def __init__(self, stub, spec_vocabs):
+    def __init__(self, stub, new_words):
         self.stub = stub
-        self.spec_vocabs = spec_vocabs
+        self.new_words = new_words
 
-    def get_word_group_ids (self):
-        # TODO: retrieve the WordGroups we have stored from registry. 
-        # return them in an appropriate format for easy comparison 
-        return
-    def find_word_group (self):
-        # TODO: using the list of IDs returned by get_word_group_ids, 
-        # try to find the closest wordgroup for each spec vocab.
-        # if successful, save it in the format vocab:id  else vocab :None
-        # return a list of these
-        return
+    def get_word_group_ids(self):
+        # TODO: retrieve the WordGroups we have stored from registry.
+        # return them in an appropriate format for easy comparison
+        return []
+
+    # format: {word : (cluster_id, distance)}
+    def find_word_groups(self):
+        try:
+            ids = self.get_word_group_ids()
```

Comment:
This can potentially return WordGroup objects and not only ids, so that we can reuse those objects later while constructing the Variations. We can revisit this later while implementing the variation part.